### PR TITLE
Sam/account refactor

### PR DESCRIPTION
**Pull request by kels0la**
Account page refactor basic refactor setup. Needs styling/beautification, some more data put into transaction data, and a password reset.

### CODE REVIEW

File: client/src/pages/Account/AccountPage.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { MainWrapper, Button } from '../../components/Widgets';
+import Sidebar from '../../components/Sidebar/Sidebar';
+import AccountViewConductor from './AccountViewConductor';
+import { DraftListItem } from '../../components/Drafts/DraftListItem';
+import List from '../../components/Widgets/List';
+
```
Comment (robbyphillips) [inline client/src/pages/Account/AccountPage.js]: For this component, we can clean this up a bit by pulling off a few props, the grabbing all of leftovers with the `...` rest operator.
```
const Account Page = props => {
  const { switchView, drafts, removeDraft, ...conductorProps } = props
  return (
    // ...
    <AccountViewConductor {...conductorProps} />
    // ...
)
```

File: client/src/pages/Account/AccountPage.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          onChange={props.onChange}
+          handleFormSubmit={props.handleFormSubmit}
+          handleMoneyBtnIdSubmit={props.handleMoneyBtnIdSubmit}
+        />
+      </MainWrapper>
+      <MainWrapper styles='w-1/5'>
+        <Sidebar>
+          <h2 className='font-header text-center'>Your drafts</h2>
+          <hr className='h-px'/>
+            <DraftsList data={props.drafts} removeDraft={this.removeDraft} />
```
Comment (robbyphillips) [inline client/src/pages/Account/AccountPage.js]: if we do the thing above
```suggestion
            <DraftsList data={drafts} removeDraft={removeDraft} />
```
or
```suggestion
            <DraftsList data={props.drafts} removeDraft={props.removeDraft} />
```

File: client/src/pages/Account/AccountPage.js (hunk 1 above)
Comment (kels0la) [reply client/src/pages/Account/AccountPage.js]: Updated

File: client/src/pages/Account/AccountPage.js (hunk 2 above)
Comment (kels0la) [reply client/src/pages/Account/AccountPage.js]: SWAPEROOSKI

**Comment by kels0la** [comment]
Great suggestions and I have now pushed them in. I'll let you do the honors of merging the pull request.

**Pull request merged**